### ISSUE TITLE
Any plans to support react-router v6

### ISSUE THREAD

**Issue by gitter-me** [xiel/use-location-state#230]
Hi, are there any plans to support [react-router v6](https://reactrouter.com/docs/en/v6)?

The current plugin for v5 show this error with v6:
 ```ts node_modules/react-router-use-location-state/dist/react-router-use-location-state.esm.js:3:9: 
  error: No matching export in "node_modules/react-router/index.js" for import "__RouterContext"
    3 │ import { __RouterContext } from 'react-router';

**Comment by xiel** [comment]
Yes, I'll try to support it soon, but it might take a few days.

**Comment by xiel** [comment]
Hey @gitter-me 

support for react-router v6 is released with [v3.0.0](https://github.com/xiel/use-location-state/releases/tag/v3.0.0)

Setting two states at the same time is supported, just call them right after each other and they will be automatically merged.
```
setName('Kim')
setAge(45)
```

Cheers and thanks for opening the issue.
